blocnotes/Note.h

```diff
@@ -0,0 +1,13 @@
+//
```

Comment:
We might want to git remove this file, as we don't use it and it just clutters up the filesystem, at the moment. Same goes for its `.m` file.